docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,302 @@
+.. role:: hidden
+    :class: hidden-section
+
+Gender Classification and Eyes Location Detection: A Two Task Problem
+*********************************************************************
+
+.. note::
+
+    - See the notebook `here <https://github.com/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+    - Run in `Google Colab <https://colab.research.google.com/github/GRAAL-Research/poutyne/blob/master/examples/classification_and_regression.ipynb>`_
+
+In this example, we are going to implement a multi-task problem. We try to identify the gender of the people, as well as locating their eyes in the image. Hence, we have two different tasks: classification (to identify the gender) and regression (to find the location of the eyes). We are going to use a single network (A CNN) to perform both tasks, however, we will need to apply different loss functions, each proper to a specific task.
+
+Let's import all the needed packages.
+
+.. code-block:: python
+
+    import math
+    import os
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import wget
+    import zipfile
+    import cv2
+    import torch
+    import torch.nn as nn
+    import torch.nn.functional as F
+    import torch.optim as optim
+    from torch.autograd import Variable
+    import torchvision.datasets as datasets
+    import torchvision.models as models
+    import torchvision.transforms as tfms
+    from poutyne import set_seeds, Model, ModelCheckpoint, CSVLogger, Experiment, StepLR
+    from torch.utils.data import DataLoader, Subset, Dataset
+    from torchvision.utils import make_grid
+
+Training Constants
+==================
+
+.. code-block:: python
+
+    num_epochs = 15
+    learning_rate = 0.001
+    batch_size = 32
+    image_size = 224
+    valid_split_percent = 0.1
+    w, h = 218, 178   # the width and the hight of original images before resizing
+    momentum = 0.5
+    set_seeds(42)
+    imagenet_mean = [0.485, 0.456, 0.406]  # mean of the ImageNet dataset for normalizing 
+    imagenet_std = [0.229, 0.224, 0.225]  # std of the ImageNet dataset for normalizing
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    print('The running processor is...', device) 
+
+CelebA Dataset
+==============
+
+The section below consists of a few lines of codes that help us download celebA dataset from a public web source and unzip them. Downloading the Celeba dataset can be also done directly by `torch.datasets.CelebA(data_root, download =  True)`. however, due to the high traffic of Google Drive ( the main source of the dataset) it usually fails to function. hence we decided to download it from another public source but use it with `torch.datasets.CelebA()`.
+
+Fetching data 
+=============
+
+The section below consists of a few lines of codes that help us download celebA dataset from a public web source and unzip them.
```

Comment:
This repeats what has just been said.

Reply:
I didn't get your point. Do you mean I have repeated it in another place?

Reply:
Yeah, the text above also says that.